tokens-filter/src/json.rs

```diff
@@ -0,0 +1,59 @@
+use std::sync::Arc;
+
+use async_trait::async_trait;
+use rust_utils::tokens::get_token_symbol_by_mint_from_json;
+use solana_sdk::pubkey::Pubkey;
+
+use crate::CheckToken;
+
+pub struct JsonChecker;
+
+#[async_trait]
+impl CheckToken for JsonChecker {
+    type Token = Pubkey;
+
+    #[tracing::instrument(skip(self), err)]
+    async fn check_token(&self, token: &Self::Token) -> anyhow::Result<bool> {
+        match get_token_symbol_by_mint_from_json(&token.to_string()).await {
+            Ok(_) => Ok(true),
+            Err(e) => {
+                if e.to_string().contains("token not found") {
+                    Ok(false)
+                } else {
+                    Err(e)
+                }
+            },
+        }
+    }
+}
+
+#[async_trait]
+impl CheckToken for Arc<JsonChecker> {
```

Comment:
This code is not necessary. `JsonChecker` is empty struct

Reply:
done